src/quill.imageDrop.ts

```diff
@@ -72,15 +72,27 @@ export class ImageDrop {
   }
 
   private async handlePaste(evt: ClipboardEvent) {
-    await this.handleDataTransferList(evt.clipboardData?.items, evt);
+    if (evt.clipboardData) {
+      await this.handleDataTransferList(evt.clipboardData!, evt);
+    }
   }
 
-  private async handleDataTransferList(dataTransferItems: DataTransferItemList | undefined, evt: { preventDefault: () => void }) {
-    const items = Array.from(dataTransferItems || []);
+  private async handleDataTransferList(dataTransfer: DataTransfer, evt: Event) {
```

Comment:
Probably need to change the method name to reflect the arguments a bit better

```suggestion
  private async handleDataTransfer(dataTransfer: DataTransfer | undefined, evt: Event) {
```